package.json

```diff
@@ -45,7 +45,14 @@
     "test": "yarn generate test && yarn test:flow && yarn test:lint && yarn test:render && yarn test:unit && yarn test:validate-xml"
   },
   "dependencies": {
+    "@react-navigation/bottom-tabs": "6.5.7",
```

Comment:
I think we might want to make these peer-dependencies in case we don't get to inject these before we release